framework/branch-key-store.md

```diff
@@ -427,11 +427,11 @@ If the Keystore's KMS Configuration is `Discovery` or `MRDiscovery`,
 this operation MUST immediately fail.
 
 VersionKey MUST first get the active version for the branch key from the keystore
-by calling the configured [EncryptedKeyStore interface's](./key-store/encrypted-key-store.md#interface)
-[GetEncryptedActiveBranchKey](./key-store/encrypted-key-store.md##getencryptedactivebranchkey)
+by calling the configured [KeyStorage interface's](./key-store/key-storage.md#interface)
+[GetEncryptedActiveBranchKey](./key-store/key-storage.md##getencryptedactivebranchkey)
```

Comment:
(super nit)
```suggestion
[GetActiveEncryptedBranchKey](./key-store/key-storage.md##getactiveencryptedbranchkey)
```
Most importantly aligns with your structure for `WriteNewEncryptedBranchKey` but it's also grammatically better

Reply:
Talked.

In English because the order of additives should be, active; encrypted as opposed to encrypted; active.
However, the upstream API (the key store) is asking for the `ActiveBranchKey` and `BranchKeyVersion`. This is why the `Encrypted` leads. The thing being returned in the `ActiveBranchKey`. This is because the `BranchKey` is a logical construct that is represented by the identify.
Leaving the API and merging.